server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/AnalyzeVisitor.java

```diff
@@ -350,6 +357,11 @@ public Analysis visitQuery(QueryStatement queryStatement, MPPQueryContext contex
     return analysis;
   }
 
+  private Analysis finishQuery(Analysis analysis) {
+    analysis.setFinishQueryAfterAnalyze(true);
```

Comment:
We need to set empty headers in different formats.
```suggestion
if (queryStatement.isSelectInto()) {
          analysis.setRespDatasetHeader(
            DatasetHeaderFactory.getSelectIntoHeader(queryStatement.isAlignByDevice()));
        }
        if (queryStatement.isLastQuery()) {
       analysis.setRespDatasetHeader(DatasetHeaderFactory.getLastQueryHeader());
        }
    analysis.setFinishQueryAfterAnalyze(true);
```